### ISSUE TITLE
[feature request] Export @values from another file

### ISSUE THREAD

**Issue by getkey** [tivac/modular-css#354]
Basically, something equivalent to the ES6 code:
```js
export * from './other_file.js';
```

Here is an example of a possible use case, as you requested in our Gitter conversation:

```css
/* common_vars.css */
@value primary: blue;
```

```css
/* mobile_vars.css */
@value * from './common_vars.css';

@value mw: 100%;
```

```css
/* desktop_vars.css */
@value * from './common_vars.css';

@value mw: 1024px;
```

```css
/* desktop-component.css */
value * as v from "./desktop_vars.css";
.wrapper {
    max-width: v.mw;
    color: v.primary;
}
```

```css
/* mobile-component.css */
value * as v from "./mobile_vars.css";
.wrapper {
    max-width: v.mw;
}
.button {
    color: v.primary;
}
```

Having this would allow us to share some variables among all component, and some between only a subset of our components. It is currently possible to do this by importing values from multiple files but this addition would make it more generic and much nicer in more complicated situations.

## Our specific use case

In our case we made a theme engine that makes it possible for us to combine, for a given class name of a given file in a given theme (all our themes are in a specific directory), all the rules defined for this class name in the hierarchy of themes.

Here is a diagram of our themes:

```
Boxes with `-`: themes that serve as the basis for another theme.
Boxes with `=`: themes that are exported as a .css file.

                +--------+
         +------+ common +------------+
         |      +--------+            |
         |                            |
         |                            |
         |                            |
         |                            |
         |                            |
+========v=======+               +----v----+
| default-mobile |         +-----+ desktop +------+
+=======+========+         |     +---------+      |
        |                  |                      |
        |                  |                      |
        |                  |                      |
        |                  |                      |
        |                  |                      |
 +======v======+       +===v==+              +====v====+
 | kids-mobile |       | kids |              | default |
 +=============+       +======+              +=========+
```

It is very difficult to share variable among themes because of this hierarchy, and having the ability to export `@values` from other files would makes this much better: we could have a `vars.css` file for each theme. Each of these would import and then export variables from the parent theme (if any).

**Comment by tivac** [comment]
Sorry, I haven't forgotten this. Been busy w/ life and settling into a new job. Hoping to get to it sometime in the next week or two.

After some pondering I think allowing for re-exporting values w/ specific overrides seems pretty reasonable to me. Not 100% sure about that syntax but I'll get back to you after I've prototyped something and we'll see what feels good.

**Comment by getkey** [comment]
Awesome, looking forward to it!
Hope all is well for you :-)

**Comment by tivac** [comment]
Fixed by 5c2af28a8bf7f871d5d427fb32cd398e38d7c521

**Comment by tivac** [comment]
This is published as `6.1.0`.
